autograph.yaml

```diff
@@ -900,6 +900,129 @@ signers:
         mwIDAQAB
         -----END PUBLIC KEY-----
 
+    - id: webextensions-rsa-with-recommendation
+      type: xpi
+      # The signing parameters for each type of add-on are
+      # 'add-on' are signed with the OU 'Production' and the provided ID
+      # 'extension' are signed with the OU 'Mozilla Extensions' and the provided ID
+      # 'system add-on' are signed with the OU 'Mozilla Components' and the provided ID
+      # 'hotfix' are signed with the OU 'Production' and the ID 'firefox-hotfix@mozilla.org'
+      mode: add-on-with-recommendation
+      # RSA key gen is slow and CPU intensive, so we can optionally
+      # pregenerate and cache keys with a worker pool
+      recommendation:
+        path: "test-recommendation.json"
```

Comment:
perhaps this shouldn't be configurable?

Reply:
Depending on whether the proposed filename is used in a lot of existing XPIs my plan was to allow AMO to change it without requiring a code change. :woman_shrugging: